gym_ai2thor/tasks.py

```diff
@@ -2,8 +2,10 @@
 Different task implementations that can be defined inside an ai2thor environment
 """
 from collections import Counter
+import random
 
 from gym_ai2thor.utils import InvalidTaskParams
+from gym_ai2thor.task_utils import get_word_to_idx, check_if_focus_and_close_enough_to_object_type
 
 
 class TaskFactory:
```

Comment:
let's get rid of this task entirely by calling `getattr(gym_ai2thor.tasks, config['task']['task_name'])(**config)` from AI2ThorEnv